themes/janeswalk/js/components/pages/WalkPage/WalkDescription.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const WalkDescription = ({longDescription}) => (
+  <section className="walkDescription">
+    <a name="About This Walk"></a>
+    <h2>About This Walk</h2>
+    <article dangerouslySetInnerHTML={{__html: longDescription}}></article>
```

Comment:
Let's keep this for now, but I'm thinking it might make sense to load the walkdescription as markdown, or perhaps do something on a flux store to sanitize the HTML so only allowed elements can be loaded. `dangerouslySetInnerHTML` is a blunt instrument, but if there were a way to parse out all the `<b> <strong> <em> <i>`, and ignore everything else (including the `<a>` and `<script>` tags), 

Obviously this will never be less secure than rendering that description directly as HTML (and is probably a bit better, really), but with more and more of the site moving into client-side templates, we want as few places able to run `<script>` directly as possible.